reporting/sensitivity-labels.md

```diff
@@ -0,0 +1,39 @@
+---
+description: The Sensitivity Labels report provides an overview of your sensitivity labels.
+---
+
+# Sensitivity Labels Report
+
+This article contains information on the Sensitivity Labels report available through the **Report Center**:
+
+The Sensitivity Labels report provides a complete overview of your Sensitivity Labels and summarized data about your workspaces and files. 
+
+Use this report to ensure sensitive data is properly labeled and assigned to the correct number of resources, identify gaps in data security, and maintain compliance with company policies. 
+
+### Generate Report
+
+ * **Click the Reports button** located on the left side of the screen.
+* **Select the Sensitivity Labels category in the filter** you can find in the upper left corner.
+* **Click the Sensitivity Labels report tile** to be redirected to the Selection screen.
+
+After clicking the **Sensitivity Labels** report tile, the Sensitivity Labels report is generated, which contains the summarized data. 
+
+### Report Data
+
+The default columns displayed in the report are:
+      
+* **Name** of the sensitivity labels
+   * The two available are **Confidential** and **Disabled**
```

Comment:
ovo je tako na našem tenantu. ali sens. labela može biti više/manje i mogu imati drugačije nazive, ovisno o tome kako ih admin postavi u tenantu.
tako da ovu točku maknuti najbolje